### PR TITLE
[tests-only][full-ci] adding test for listing drive items with new metadata in project space

### PR DESCRIPTION
**Pull request by ghost**
## Description

This PR adds test for checking the metadata of audio and image files in project pace
Scenarios added:
```cucumber
Scenario: GET extracted properties of an audio file (Project space)
Scenario: GET extracted properties of an image file (Project space)
```

## Related Issue

- Related issue: https://github.com/owncloud/ocis/issues/9180

## How Has This Been Tested?

- test environment:
- test case 1:locally
- test case 2:ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSearchContent/extractedProps.feature [hunk 1]
```diff
@@ -255,3 +255,139 @@ Feature: propfind extracted props
           }
         }
       """
+
+
+  Scenario: GET extracted properties of an audio file (Project space)
+    Given using spaces DAV path
```
Comment (anon-pradip) [inline tests/acceptance/features/apiSearchContent/extractedProps.feature:261]:
```suggestion
    Given using spaces DAV path
```

This has already been mentioned in the `Background`. So I think we can remove this from here.

Comment (anon-pradip) [inline tests/acceptance/features/apiSearchContent/extractedProps.feature:261]: This has already been mentioned in the `Background`. So I think we can remove this from here.

File: tests/acceptance/features/apiSearchContent/extractedProps.feature [hunk 2]
```diff
@@ -255,3 +255,139 @@ Feature: propfind extracted props
...
+                }
+              }
+            }
+          }
+        }
+      """
+
+
+  Scenario: GET extracted properties of an image file (Project space)
+    Given using spaces DAV path
```
Comment (anon-pradip) [inline tests/acceptance/features/apiSearchContent/extractedProps.feature:312]: Same here also. Already mentioned in the `Background`.

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**